<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Mutation Predictions</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.33.1&nbsp;&nbsp;revision 8505477f25b3
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<!--Mutation Predictions -->
<p>
<!--Output Html_Mutation_Table_String-->
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="6" align="left" class="mutation_header_row">Predicted mutations</th></tr><tr>
<th>evidence</th>
<th>position</th>
<th>mutation</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>

<!-- Item Lines -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/SNP_1.html">RA</a></td><!-- Evidence -->
<td align="right">739,687</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="center"><font class="snp_type_synonymous">H48H</font>&nbsp;(CA<font class="mutation_in_codon">T</font>&rarr;CA<font class="mutation_in_codon">C</font>)&nbsp;</td>
<td align="center"><i>Pfl01_0628</i>&nbsp;&rarr;</td>
<td align="left">conserved hypothetical protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/SNP_2.html">RA</a></td><!-- Evidence -->
<td align="right">818,804</td><!-- Position -->
<td align="center">G&rarr;C</td><!-- Cell Mutation -->
<td align="center"><font class="snp_type_nonsynonymous">G157A</font>&nbsp;(G<font class="mutation_in_codon">G</font>T&rarr;G<font class="mutation_in_codon">C</font>T)&nbsp;</td>
<td align="center"><i>Pfl01_0692</i>&nbsp;&rarr;</td>
<td align="left">putative GGDEF domain signalling&#8209;related membrane protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/MC_PLOT_65.html">MC</a>&nbsp;<a href="evidence/JC_76.html">JC</a></td><!-- Evidence -->
<td align="right">819,473</td><!-- Position -->
<td align="center">&Delta;15&nbsp;bp</td><!-- Cell Mutation -->
<td align="center">coding&nbsp;(1139&#8209;1153/1269&nbsp;nt)</td>
<td align="center"><i>Pfl01_0692</i>&nbsp;&rarr;</td>
<td align="left">putative GGDEF domain signalling&#8209;related membrane protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/MC_PLOT_66.html">MC</a>&nbsp;<a href="evidence/JC_78.html">JC</a></td><!-- Evidence -->
<td align="right">1,220,059</td><!-- Position -->
<td align="center">&Delta;42&nbsp;bp</td><!-- Cell Mutation -->
<td align="center">coding&nbsp;(889&#8209;930/1623&nbsp;nt)</td>
<td align="center"><i>Pfl01_1052</i>&nbsp;&rarr;</td>
<td align="left">putative methyl&#8209;accepting chemotaxis protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/SNP_5.html">RA</a></td><!-- Evidence -->
<td align="right">1,220,330</td><!-- Position -->
<td align="center">T&rarr;G</td><!-- Cell Mutation -->
<td align="center"><font class="snp_type_nonsynonymous">L387R</font>&nbsp;(C<font class="mutation_in_codon">T</font>G&rarr;C<font class="mutation_in_codon">G</font>G)&nbsp;</td>
<td align="center"><i>Pfl01_1052</i>&nbsp;&rarr;</td>
<td align="left">putative methyl&#8209;accepting chemotaxis protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/SNP_6.html">RA</a></td><!-- Evidence -->
<td align="right">1,224,300</td><!-- Position -->
<td align="center">G&rarr;A</td><!-- Cell Mutation -->
<td align="center"><font class="snp_type_nonsynonymous">E341K</font>&nbsp;(<font class="mutation_in_codon">G</font>AA&rarr;<font class="mutation_in_codon">A</font>AA)&nbsp;</td>
<td align="center"><i>Pfl01_1056</i>&nbsp;&rarr;</td>
<td align="left">putative chemotaxis&#8209;related two&#8209;component system, sensor kinase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/SNP_7.html">RA</a></td><!-- Evidence -->
<td align="right">1,227,186</td><!-- Position -->
<td align="center">T&rarr;A</td><!-- Cell Mutation -->
<td align="center"><font class="snp_type_nonsynonymous">L186Q</font>&nbsp;(C<font class="mutation_in_codon">T</font>G&rarr;C<font class="mutation_in_codon">A</font>G)&nbsp;</td>
<td align="center"><i>Pfl01_1058</i>&nbsp;&rarr;</td>
<td align="left">putative chemotaxis&#8209;related, two component system response regulator</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/DEL_8.html">RA</a></td><!-- Evidence -->
<td align="right">2,186,046</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="center">coding&nbsp;(126/195&nbsp;nt)</td>
<td align="center"><i>Pfl01_1912</i>&nbsp;&larr;</td>
<td align="left">carbon storage regulator</td>
</tr>
<!-- End Table Row -->
</table>
<p>
<table border="0" cellspacing="1" cellpadding="3" width="100%">
<tr><th colspan="11" align="left" class="missing_coverage_header_row">Unassigned missing coverage evidence</th></tr>
<tr><th>&nbsp;</th><th>&nbsp;</th><th>&nbsp;</th><th>seq&nbsp;id</th>
<th>start</th>
<th>end</th>
<th>size</th>
<th>&larr;reads</th>
<th>reads&rarr;</th>
<th>gene</th>
<th width="100%">description</th>
</tr>
<tr>
<td><a href="evidence/MC_SIDE_1_64.html">*</a></td>
<td><a href="evidence/MC_SIDE_2_64.html">*</a></td>
<td><a href="evidence/MC_PLOT_64.html">&divide;</a></td>
<td>CP000094</td>
<td align="right">111698</td>
<td align="right">111715</td>
<td align="right">18</td>
<td align="center">13&nbsp;[2]</td>
<td align="center">[1]&nbsp;13</td>
<td align="center"><i>Pfl01_0109/Pfl01_0110</i></td>
<td align="left">putative exported protein/conserved hypothetical protein</td>
</tr>
<tr>
<td><a href="evidence/MC_SIDE_1_67.html">*</a></td>
<td><a href="evidence/MC_SIDE_2_67.html">*</a></td>
<td><a href="evidence/MC_PLOT_67.html">&divide;</a></td>
<td>CP000094</td>
<td align="right">2106906</td>
<td align="right">2106925</td>
<td align="right">20</td>
<td align="center">11&nbsp;[0]</td>
<td align="center">[2]&nbsp;8</td>
<td align="center"><i>pvdE/Pfl01_1851</i></td>
<td align="left">putative ABC transporter ATP&#8209;binding protein/conserved hypothetical protein</td>
</tr>
<tr>
<td><a href="evidence/MC_SIDE_1_68.html">*</a></td>
<td><a href="evidence/MC_SIDE_2_68.html">*</a></td>
<td><a href="evidence/MC_PLOT_68.html">&divide;</a></td>
<td>CP000094</td>
<td align="right">2248104</td>
<td align="right">2248187</td>
<td align="right">84</td>
<td align="center">5&nbsp;[3]</td>
<td align="center">[3]&nbsp;4</td>
<td align="center"><i>Pfl01_1971</i></td>
<td align="left">putative glycine rich protein</td>
</tr>
<tr>
<td><a href="evidence/MC_SIDE_1_69.html">*</a></td>
<td><a href="evidence/MC_SIDE_2_69.html">*</a></td>
<td><a href="evidence/MC_PLOT_69.html">&divide;</a></td>
<td>CP000094</td>
<td align="right">2971844</td>
<td align="right">2971869</td>
<td align="right">26</td>
<td align="center">5&nbsp;[3]</td>
<td align="center">[1]&nbsp;4</td>
<td align="center"><i>Pfl01_2592/Pfl01_2593</i></td>
<td align="left">sigma&#8209;54 interacting regulatory protein/putative exported protein</td>
</tr>
<tr>
<td><a href="evidence/MC_SIDE_1_70.html">*</a></td>
<td><a href="evidence/MC_SIDE_2_70.html">*</a></td>
<td><a href="evidence/MC_PLOT_70.html">&divide;</a></td>
<td>CP000094</td>
<td align="right">4060219</td>
<td align="right">4060230</td>
<td align="right">12</td>
<td align="center">6&nbsp;[0]</td>
<td align="center">[3]&nbsp;5</td>
<td align="center"><i>cspD/icd</i></td>
<td align="left">cold shock&#8209;like protein CspD/isocitrate dehydrogenase, NADP</td>
</tr>
<tr>
<td><a href="evidence/MC_SIDE_1_71.html">*</a></td>
<td><a href="evidence/MC_SIDE_2_71.html">*</a></td>
<td><a href="evidence/MC_PLOT_71.html">&divide;</a></td>
<td>CP000094</td>
<td align="right">6158357</td>
<td align="right">6158378</td>
<td align="right">22</td>
<td align="center">4&nbsp;[3]</td>
<td align="center">[3]&nbsp;4</td>
<td align="center"><i>argH/Pfl01_5490</i></td>
<td align="left">argininosuccinate lyase/putative glutathione S&#8209;transferase domain protein</td>
</tr>
</table>
<p>
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="12" align="left" class="new_junction_header_row">Unassigned new junction evidence</th></tr>
<!-- Header Lines for New Junction -->
<tr>
<th colspan="2">&nbsp;</th>
<th>seq&nbsp;id</th>
<th>position</th>
<th>reads&nbsp;(cov)</th>
<th>reads&nbsp;(cov)</th>
<th>score</th>
<th>skew</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">product</th>
</tr>

<!-- Item Lines for New Junction -->
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_84.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_84.html">?</a></td>
<td rowspan="1" class="junction_repeat">CP000094</td>
<td align="center" class="junction_repeat">2415659&nbsp;=</td><td align="center" class="junction_repeat">NA (NA)</td><td rowspan="2" align="center">114 (2.730)</td>
<td rowspan="2" align="center">56/214</td>
<td rowspan="2" align="center">0.0</td>
<td rowspan="2" align="center">98.3%</td>
<td align="center" class="junction_repeat">noncoding&nbsp;(1/28&nbsp;nt)</td>
<td align="center" class="junction_repeat"><i>Left&nbsp;IR&nbsp;of&nbsp;the&nbsp;IS&nbsp;element;&nbsp;not&nbsp;perfect&nbsp;repeat&nbsp;of&nbsp;other&nbsp;end</i></td>
<td class="junction_repeat">Left IR of the IS element; not perfect repeat of other end</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_84.html">?</a></td><td rowspan="1" class="junction_gene">CP000094</td>
<td align="center" class="junction_gene">=&nbsp;5266677</td>
<td align="center" class="junction_gene">2 (0.050)</td><td align="center" class="junction_gene">intergenic&nbsp;(+54/&#8209;56)</td>
<td align="center" class="junction_gene"><i>Pfl01_4665/Pfl01_4666</i></td>
<td class="junction_gene">putative family S43 non&#8209;peptidase&#8209;like protein/putative GGDEF domain membrane protein</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="2"><a href="evidence/JC_85.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_85.html">?</a></td>
<td rowspan="1" class="junction_repeat">CP000094</td>
<td align="center" class="junction_repeat">=&nbsp;2416971</td><td align="center" class="junction_repeat">NA (NA)</td><td rowspan="2" align="center">83 (1.990)</td>
<td rowspan="2" align="center">44/214</td>
<td rowspan="2" align="center">0.1</td>
<td rowspan="2" align="center">97.7%</td>
<td align="center" class="junction_repeat">noncoding&nbsp;(28/28&nbsp;nt)</td>
<td align="center" class="junction_repeat"><i>Right&nbsp;IR&nbsp;of&nbsp;the&nbsp;IS&nbsp;element;&nbsp;not&nbsp;perfect&nbsp;repeat&nbsp;of&nbsp;other&nbsp;end</i></td>
<td class="junction_repeat">Right IR of the IS element; not perfect repeat of other end</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="1"><a href="evidence/JC_SIDE_2_85.html">?</a></td><td rowspan="1" class="junction_gene">CP000094</td>
<td align="center" class="junction_gene">5266675&nbsp;=</td>
<td align="center" class="junction_gene">2 (0.050)</td><td align="center" class="junction_gene">intergenic&nbsp;(+52/&#8209;58)</td>
<td align="center" class="junction_gene"><i>Pfl01_4665/Pfl01_4666</i></td>
<td class="junction_gene">putative family S43 non&#8209;peptidase&#8209;like protein/putative GGDEF domain membrane protein</td>
</tr>
</table>
</body></html>