<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Mutation Predictions</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.33.1&nbsp;&nbsp;revision 8505477f25b3
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<!--Mutation Predictions -->
<p>
<!--Output Html_Mutation_Table_String-->
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="6" align="left" class="mutation_header_row">Predicted mutations</th></tr><tr>
<th>evidence</th>
<th>position</th>
<th>mutation</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">description</th>
</tr>

<!-- Item Lines -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/SNP_1.html">RA</a></td><!-- Evidence -->
<td align="right">1,220,312</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="center"><font class="snp_type_nonsynonymous">A381V</font>&nbsp;(G<font class="mutation_in_codon">C</font>C&rarr;G<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>Pfl01_1052</i>&nbsp;&rarr;</td>
<td align="left">putative methyl&#8209;accepting chemotaxis protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/SNP_2.html">RA</a></td><!-- Evidence -->
<td align="right">1,221,196</td><!-- Position -->
<td align="center">T&rarr;C</td><!-- Cell Mutation -->
<td align="center"><font class="snp_type_nonsynonymous">L135P</font>&nbsp;(C<font class="mutation_in_codon">T</font>G&rarr;C<font class="mutation_in_codon">C</font>G)&nbsp;</td>
<td align="center"><i>Pfl01_1053</i>&nbsp;&rarr;</td>
<td align="left">putative chemotaxis&#8209;related protein</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/MC_PLOT_62.html">MC</a>&nbsp;<a href="evidence/JC_71.html">JC</a></td><!-- Evidence -->
<td align="right">1,221,330</td><!-- Position -->
<td align="center">&Delta;1,274&nbsp;bp</td><!-- Cell Mutation -->
<td align="center">coding&nbsp;(5&#8209;1278/1278&nbsp;nt)</td>
<td align="center"><i>Pfl01_1054</i>&nbsp;&rarr;</td>
<td align="left">putative chemotaxis&#8209;related methyltransferase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/INS_4.html">RA</a></td><!-- Evidence -->
<td align="right">1,224,117</td><!-- Position -->
<td align="center">+G</td><!-- Cell Mutation -->
<td align="center">coding&nbsp;(838/2295&nbsp;nt)</td>
<td align="center"><i>Pfl01_1056</i>&nbsp;&rarr;</td>
<td align="left">putative chemotaxis&#8209;related two&#8209;component system, sensor kinase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/MC_PLOT_63.html">MC</a>&nbsp;<a href="evidence/JC_72.html">JC</a></td><!-- Evidence -->
<td align="right">1,225,308</td><!-- Position -->
<td align="center">&Delta;69&nbsp;bp</td><!-- Cell Mutation -->
<td align="center">coding&nbsp;(2029&#8209;2097/2295&nbsp;nt)</td>
<td align="center"><i>Pfl01_1056</i>&nbsp;&rarr;</td>
<td align="left">putative chemotaxis&#8209;related two&#8209;component system, sensor kinase</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/DEL_6.html">RA</a></td><!-- Evidence -->
<td align="right">2,186,046</td><!-- Position -->
<td align="center">&Delta;1&nbsp;bp</td><!-- Cell Mutation -->
<td align="center">coding&nbsp;(126/195&nbsp;nt)</td>
<td align="center"><i>Pfl01_1912</i>&nbsp;&larr;</td>
<td align="left">carbon storage regulator</td>
</tr>
<!-- End Table Row -->

<!-- Print The Table Row -->
<tr class="normal_table_row">
<td align="center"><a href="evidence/SNP_7.html">RA</a></td><!-- Evidence -->
<td align="right">3,463,653</td><!-- Position -->
<td align="center">C&rarr;T</td><!-- Cell Mutation -->
<td align="center"><font class="snp_type_nonsynonymous">T242I</font>&nbsp;(A<font class="mutation_in_codon">C</font>C&rarr;A<font class="mutation_in_codon">T</font>C)&nbsp;</td>
<td align="center"><i>Pfl01_3026</i>&nbsp;&rarr;</td>
<td align="left">conserved hypothetical protein</td>
</tr>
<!-- End Table Row -->
</table>
<p>
<table border="0" cellspacing="1" cellpadding="3" width="100%">
<tr><th colspan="11" align="left" class="missing_coverage_header_row">Unassigned missing coverage evidence</th></tr>
<tr><th>&nbsp;</th><th>&nbsp;</th><th>&nbsp;</th><th>seq&nbsp;id</th>
<th>start</th>
<th>end</th>
<th>size</th>
<th>&larr;reads</th>
<th>reads&rarr;</th>
<th>gene</th>
<th width="100%">description</th>
</tr>
<tr>
<td><a href="evidence/MC_SIDE_1_61.html">*</a></td>
<td><a href="evidence/MC_SIDE_2_61.html">*</a></td>
<td><a href="evidence/MC_PLOT_61.html">&divide;</a></td>
<td>CP000094</td>
<td align="right">596003</td>
<td align="right">596115</td>
<td align="right">113</td>
<td align="center">6&nbsp;[5]</td>
<td align="center">[4]&nbsp;6</td>
<td align="center"><i>Pfl01_0513</i></td>
<td align="left">chemotaxis MotB protein</td>
</tr>
<tr>
<td><a href="evidence/MC_SIDE_1_64.html">*</a></td>
<td><a href="evidence/MC_SIDE_2_64.html">*</a></td>
<td><a href="evidence/MC_PLOT_64.html">&divide;</a></td>
<td>CP000094</td>
<td align="right">2106906</td>
<td align="right">2106925</td>
<td align="right">20</td>
<td align="center">10&nbsp;[0]</td>
<td align="center">[0]&nbsp;7</td>
<td align="center"><i>pvdE/Pfl01_1851</i></td>
<td align="left">putative ABC transporter ATP&#8209;binding protein/conserved hypothetical protein</td>
</tr>
<tr>
<td><a href="evidence/MC_SIDE_1_65.html">*</a></td>
<td><a href="evidence/MC_SIDE_2_65.html">*</a></td>
<td><a href="evidence/MC_PLOT_65.html">&divide;</a></td>
<td>CP000094</td>
<td align="right">2248102</td>
<td align="right">2248190</td>
<td align="right">89</td>
<td align="center">8&nbsp;[4]</td>
<td align="center">[4]&nbsp;6</td>
<td align="center"><i>Pfl01_1971</i></td>
<td align="left">putative glycine rich protein</td>
</tr>
<tr>
<td><a href="evidence/MC_SIDE_1_66.html">*</a></td>
<td><a href="evidence/MC_SIDE_2_66.html">*</a></td>
<td><a href="evidence/MC_PLOT_66.html">&divide;</a></td>
<td>CP000094</td>
<td align="right">2950774&#8211;2950777</td>
<td align="right">2950800</td>
<td align="right">24&#8211;27</td>
<td align="center">6&nbsp;[4]</td>
<td align="center">[5]&nbsp;7</td>
<td align="center"><i>Pfl01_2573/Pfl01_2574</i></td>
<td align="left">putative transmembrane transport protein/LysR&#8209;family transcriptional regulator</td>
</tr>
<tr>
<td><a href="evidence/MC_SIDE_1_67.html">*</a></td>
<td><a href="evidence/MC_SIDE_2_67.html">*</a></td>
<td><a href="evidence/MC_PLOT_67.html">&divide;</a></td>
<td>CP000094</td>
<td align="right">3672387</td>
<td align="right">3672442</td>
<td align="right">56</td>
<td align="center">6&nbsp;[4]</td>
<td align="center">[3]&nbsp;7</td>
<td align="center"><i>gspC</i></td>
<td align="left">putative type II secretion system membrane protein gspC</td>
</tr>
<tr>
<td><a href="evidence/MC_SIDE_1_68.html">*</a></td>
<td><a href="evidence/MC_SIDE_2_68.html">*</a></td>
<td><a href="evidence/MC_PLOT_68.html">&divide;</a></td>
<td>CP000094</td>
<td align="right">4276372</td>
<td align="right">4276380</td>
<td align="right">9</td>
<td align="center">14&nbsp;[3]</td>
<td align="center">[4]&nbsp;6</td>
<td align="center"><i>codA/Pfl01_3780</i></td>
<td align="left">cytosine deaminase/carboxymuconolactone decarboxylase</td>
</tr>
</table>
<p>
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="12" align="left" class="new_junction_header_row">Unassigned new junction evidence</th></tr>
<!-- Header Lines for New Junction -->
<tr>
<th colspan="2">&nbsp;</th>
<th>seq&nbsp;id</th>
<th>position</th>
<th>reads&nbsp;(cov)</th>
<th>reads&nbsp;(cov)</th>
<th>score</th>
<th>skew</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">product</th>
</tr>

<!-- Item Lines for New Junction -->
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_70.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_70.html">?</a></td>
<td rowspan="1" class="junction_gene">CP000094</td>
<td align="center" class="junction_gene">55317&nbsp;=</td><td align="center" class="junction_gene">45 (0.910)</td><td rowspan="2" align="center">52 (1.060)</td>
<td rowspan="2" align="center">32/208</td>
<td rowspan="2" align="center">0.3</td>
<td rowspan="2" align="center">55.1%</td>
<td align="center" class="junction_gene">coding&nbsp;(1929/2091&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>Pfl01_0050</i></td>
<td class="junction_gene">conserved GGDEF domain protein</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_70.html">?</a></td><td rowspan="1" class="junction_gene">CP000094</td>
<td align="center" class="junction_gene">=&nbsp;55829</td>
<td align="center" class="junction_gene">40 (0.820)</td><td align="center" class="junction_gene">coding&nbsp;(1417/2091&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>Pfl01_0050</i></td>
<td class="junction_gene">conserved GGDEF domain protein</td>
</tr>
</table>
</body></html>